docs/pages/configuration.rst

```diff
@@ -1,4 +1,204 @@
 Configuration
 #############
+.. highlight:: YAML
+.. rstcheck: ignore-directives=doxygenstruct
+.. rstcheck: ignore-roles=repo_file
 
-A guide on how to configure and tune wavemap for usage on your own datasets and robots is coming soon.
+Wavemap's configuration system is designed to be flexible and expressive. Let's start with an example to illustrate the overall structure:
+
+.. literalinclude:: ../../ros/wavemap_ros/config/wavemap_ouster_os0.yaml
+    :language: YAML
+    :caption: :repo_file:`Example config to use wavemap with an Ouster OS0-128 LiDAR<ros/wavemap_ros/config/wavemap_ouster_os0.yaml>`
+
+The map's general settings, such as the coordinate frame in which it's represented and the rate at which its published, are configured under ``map/general``.
```

Comment:
"which its published" -> "which it's published"

Reply:
Thanks!